### ISSUE TITLE
How To Make Maps N Apply In Bombsquad???

### ISSUE THREAD

**Issue by bilal080** [Mrmaxmeier/BombSquad-Community-Mod-Manager#44]
Bro plz make a vid in utube how to make our own maps n apply in bombsquad game plzz

**Comment by TheMikirog** [comment]
Let me get some things out of the way:
I'm not the creator of BombSquad Mod Manager. Unless you want to talk to me directly, then ok. 

I purposefully avoided making any kind of tutorials on modding for several reasons. Most of the time I encounter people who want to jump into modding and when they finally get that power, I encounter a shit-ton of crappy maps with bad lighting, migraine inducing colors and lackluster gameplay. These people are not game designers after all. I'm also not a game designer, but I took that extra step to understand what made BombSquad appealing to the eyes and of course, fun in the long run. 

Even though my mod pack encourages modding (which succeeded in the end) it created a double-edged sword along the way. Rule of thumb: if you really, really, REALLY care about modding, you would take some effort and not go the easiest route and seek advice from the masters on the basics of BombSquad modding. I and Max invented the wheel (Max by providing the community with the right tools and me by using them properly) and using that knowledge right is the hardest and most satisfying part of modding. Now learning is easier than ever, but not because I tell you what to do, but because the right tools are at your disposal. It's your job to understand the basics of them and make great work. The only thing I can help with is with minor coding stuff that isn't as obvious (this is the info I gathered from conversations with Eric).

In conclusion: learn the tools, research the code and don't go the easy route

Also, don't use your phone to create mods. EVER.
